graphs/dijkstra.py

```diff
@@ -1,75 +1,77 @@
 
-	def __init__(self, vertices):
-		self.V = vertices
-		self.graph = [[0 for column in range(vertices)]
-					for row in range(vertices)]
+    def printsolution(self, dist):
+        print("Vertex \t Distance from Source")
+        for node in range(self.V):
+            print(node, "\t\t", dist[node])
 
-	def printsolution(self, dist):
-		print("Vertex \t Distance from Source")
-		for node in range(self.V):
-			print(node, "\t\t", dist[node])
+    # A utility function to find the vertex with
+    # minimum distance value, from the set of vertices
+    # not yet included in shortest path tree
+    def mindistance(self, dist, sptset):
 
-	# A utility function to find the vertex with
-	# minimum distance value, from the set of vertices
-	# not yet included in shortest path tree
-	def mindistance(self, dist, sptset):
+        # Initialize minimum distance for next node
+        minimum = 1e7
 
-		# Initialize minimum distance for next node
-		minimum = 1e7
+        # Search not nearest vertex not in the
+        # shortest path tree
+        for v in range(self.V):
+            if dist[v] < minimum and sptset[v] == False:
+                minimum = dist[v]
+                min_index = v
 
-		# Search not nearest vertex not in the
-		# shortest path tree
-		for v in range(self.V):
-			if dist[v] < minimum and sptset[v] == False:
-				minimum = dist[v]
-				min_index = v
+        return min_index
 
-		return min_index
+    # Function that implements Dijkstra's single source
+    # shortest path algorithm for a graph represented
+    # using adjacency matrix representation
+    def dijkstra(self, src):
 
-	# Function that implements Dijkstra's single source
-	# shortest path algorithm for a graph represented
-	# using adjacency matrix representation
-	def dijkstra(self, src):
+        dist = [1e7] * self.V
+        dist[src] = 0
+        sptset = [False] * self.V
 
-		dist = [1e7] * self.V
-		dist[src] = 0
-		sptset = [False] * self.V
+        for cout in range(self.V):
 
-		for cout in range(self.V):
+            # Pick the minimum distance vertex from
+            # the set of vertices not yet processed.
+            # u is always equal to src in first iteration
+            u = self.mindistance(dist, sptset)
 
-			# Pick the minimum distance vertex from
-			# the set of vertices not yet processed.
-			# u is always equal to src in first iteration
-			u = self.mindistance(dist, sptset)
+            # Put the minimum distance vertex in the
+            # shortest path tree
+            sptset[u] = True
 
-			# Put the minimum distance vertex in the
-			# shortest path tree
-			sptset[u] = True
+            # Update dist value of the adjacent vertices
+            # of the picked vertex only if the current
+            # distance is greater than new distance and
+            # the vertex in not in the shortest path tree
+            for v in range(self.V):
+                if (
```

Comment:
Change this if statement like the other comment above as well so that you won't have to equate anything to `False`. Avoid using `False`. Use `not` `<false condition>` instead.